### PR TITLE
Feature/25953 add css processing

### PR DESCRIPTION
**Pull request by timwright12**
## Description

Releasing to dev first

https://github.com/department-of-veterans-affairs/va.gov-team/issues/25953

## Testing Done
Running commands: `yarn build` for each environment, clean CI, and examined all S3 uploads. Will also test post-merge and deployment for issues.

Also adding webpack into the watch process.

### CODE REVIEW

**Comment by timwright12** [comment]
Pausing this one until a11y issues in CI are fixed

File: src/site/includes/header.html [hunk 1]
```diff
@@ -61,6 +61,9 @@
 
   <!-- CSS -->
   <link rel="stylesheet" data-entry-name="style.css">
+  {% if buildtype != 'vagovdev' or buildtype != 'localhost' %}
```
Comment (timwright12) [inline src/site/includes/header.html:64]: releasing to dev and local for now

File: src/site/includes/header.html [hunk 2]
```diff
@@ -61,6 +61,9 @@
 
   <!-- CSS -->
   <link rel="stylesheet" data-entry-name="style.css">
+  {% if buildtype == 'vagovdev' or buildtype == 'localhost' %}
```
Comment (kelsonic) [inline src/site/includes/header.html:64]: should we add preview server to this list?

Comment (timwright12) [reply src/site/includes/header.html:64]: If you want, it'll probably only be there until I confirm it works in dev

**Review by kelsonic — APPROVED** [review]

**Review by ndsprinkle — APPROVED** [review]

**Comment by mpelzsherman** [comment]
This looks great! Have we added any CSS to content-build to test?

**Comment by timwright12** [comment]
@mpelzsherman I've added it locally, but I was having a hard time getting CSS that's clean in vets-website, even stuff that's only meant for the homepage is used elsewhere

**Pull request merged**